mod-source-record-manager-server/src/main/java/org/folio/services/ChangeEngineServiceImpl.java

```diff
@@ -333,6 +337,16 @@ public static boolean isStatus(HttpClientResponse response, HttpStatus status) {
     return journalRecords;
   }
 
+  private JournalRecord.EntityType inferJournalRecordEntityType(Record record) {
+    switch (record.getRecordType()) {
+      case EDIFACT:
+        return JournalRecord.EntityType.INVOICE;
```

Comment:
Hello @jeremythuff, could you please expand [the schema of `EntityType` enum](https://github.com/folio-org/data-import-raml-storage/blob/e48acdc3f446dd40ba7d9c4743b392f8a4b7a757/schemas/mod-source-record-manager/journalRecord.json#L24) with `EDIFACT` value to indicate that EDIFACT record was created

Reply:
yes I can

Reply:
thank you!